### ISSUE TITLE
Marker clicks are triggered multiple times after re-render

### ISSUE THREAD

**Issue by schneck** [YUzhva/react-leaflet-markercluster#19]
Hi there,

my render function looks like this:

```
        <MarkerClusterGroup
          markers={markers} // Simple marker array
          wrapperOptions={{ enableDefaultStyle: true, removeDuplicates: true }}
          onMarkerClick={marker => {
              console.log('click');
              emitSomeReduxAction(); // changes URL, leads to re-render
          }}
        />
```

When clicked first, the console output is a single "click". When the component re-rendered, another marker click leads to two "click" outputs. Next time, it doubles (four times, eight times) etc. It seems like there was some event handler attached again and again. 

Any idea?

**Comment by YUzhva** [comment]
yeah, looks like the bug.
I'll investigate it.

**Comment by YUzhva** [comment]
@schneck 

Fixed in: [release] 1.1.5: Fix multiple initializations of event listener (#20)

**Comment by schneck** [comment]
Confirmed, works great now - thanks for this nice project and your quick help!
